code/game/machinery/poolcontroller.dm

```diff
@@ -190,26 +195,16 @@
 
 	name = "Sea Controller"
 	desc = "A controller for the underwater portion of the sea. Players shouldn't see this."
-	deep_water = 1		//deep sea is deep water
+	var/numturf = 0
```

Comment:
Put this below deep_water, new vars should be physically at the bottom of the definition (or at least not mixed with overrides)